datasets/functionelegebieden/dataset.json

```diff
@@ -0,0 +1,70 @@
+{
+  "type": "dataset",
+  "id": "functioneleGebieden",
+  "version": "0.0.1",
+  "status": "beschikbaar",
+  "title": "Functionele gebieden",
+  "description": "Begrensd en benoemd gebied dat door een functionele eenheid beschreven wordt.",
+  "owner": "Gemeente Amsterdam",
+  "authorizationGrantor": "Onderzoek en Statistiek",
+  "creator": "bronhouder onbekend",
```

Comment:
Dit is iets dat de validator ook niet goed vindt. Zou je hier de echte naam van de bronhouder in willen vullen. Op deze manier willen van alle datasets zo compleet mogelijk alle bronhouders in beeld gaan krijgen.